### ISSUE TITLE
Accessibility: AlertDialog not accessible

### ISSUE THREAD

**Issue by ialokim** [JetBrains/compose-jb#2120]
```kotlin
@OptIn(ExperimentalMaterialApi::class)
fun main() = singleWindowApplication {
    AlertDialog(
        onDismissRequest = {},
        buttons = { Button(onClick = {}) { Text("OK") } },
        title = { Text("AlertDialog") }
    )
}
```

On macOS with VoiceOver, nothing inside `AlertDialog` is read out.

**Comment by ialokim** [comment]
Same applies to `DropDownMenu`

**Comment by ialokim** [comment]
Related to/duplicate of #2041

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2120">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
